### ISSUE TITLE
Missing counties in the map

### ISSUE THREAD

**Issue by fatema2906** [trademapper/trademapper-js#86]
There are missing countries in the map 
how can i display them?
Such as Bahrain, Singapore, and Hong kong

**Comment by martinburchell** [comment]
@fatema2906 are you saying that you have trade data for those states and they aren't being displayed? The map data are derived from https://github.com/nvkelso/natural-earth-vector

**Comment by fatema2906** [comment]
@martinburchell  yes, these countries are missing in the worldmap.js file and I couldn't find their arcs values to display them, your help will be greatly appreciated

**Comment by martinburchell** [comment]
@fatema2906 I don't know the code well enough to give you a quick answer. It looks like the map is generated by the `extract_world.sh` script. I'd start by looking there,

It doesn't look like TradeMapper is in active development at the moment. It was developed by [Aptivate](https://www.aptivate.org/) for [Traffic](https://www.traffic.org/). If you'd like Aptivate to investigate this further, please contact us via our website and we can discuss costs.

**Comment by fatema2906** [comment]
@martinburchell Can you provide me the website link please
